docs/build-linux.md

```diff
@@ -1,6 +1,185 @@
-# Build instructions
+# Building on Linux
+
+Two build methods are available for Linux - using the [system libraries](#building-using-system-libraries)
+provided by the Linux distribution or using the [vcpkg tool](#building-using-vcpkg)
+to fetch and compile dependencies.
+
+The vcpkg method is used by the team to provide the official binaries, which are
+intended to be run on different distros, without a need to install uncommon
+dependencies.
```

Comment:
Maybe clarify this: the Linux binaries only depend on glibc and that's it (if I'm not mistaken).

Reply:
OK, changed. Checked with ldd - indeed, it only depends on a few libraries from the glibc package.